src/prompt.ts

```diff
@@ -122,7 +133,10 @@ async function runOrReviseFlow(script: string, key: string) {
     message: nonEmptyScript ? 'Run this script?' : 'Revise this script?',
     options: [
       ...(nonEmptyScript
-        ? [{ label: '✅ Yes', value: 'yes', hint: 'Lets go!' }]
+        ? [
+          { label: '✅ Yes', value: 'yes', hint: 'Lets go!' },
+          { label: '📝 Edit', value: 'edit', hint: 'Make some adjustments before running' },
+        ]
         : []),
       {
         label: '📝 Revise',
```

Comment:
perhaps we use this emoji for revise and leave the edit emoji with the pencil symbol:

```suggestion
        label: '🔁 Revise',
```